contrib/wordlyworks/main.py

```diff
@@ -0,0 +1,46 @@
+"""
+Use named entity recognition to extract entities from text description (e.g. Spacy)
+Download SVGs from https://thenounproject.com/
+Ask large language model to arrange
+Generate combined SVG
+Run through e.g. Stable Diffusion and/or ask large language model to generate animation code
+"""
+
+from pprint import pformat
+from requests_oauthlib import OAuth1
+import os
+import requests
+from contrib.wordlyworks import ner.ner
+
+from dotenv import load_dotenv
+from loguru import logger
+import ner
+#from TheNounProjectAPI import API
+
+
+load_dotenv()
+
+
+def main():
+    key = os.environ["NOUN_PROJECT_KEY"]
+    secret = os.environ["NOUN_PROJECT_SECRET"]
+    logger.info(f"{key=} {secret=}")
+
+
+    auth = OAuth1(key, secret)
+    endpoint = "https://api.thenounproject.com/v2/icon/1"
+
+    response = requests.get(endpoint, auth=auth)
+    logger.info(f"response=\n{pformat(response.json())}")
+
+    if 0:
```

Comment:
Remove this

Reply:
@abrichr just a question. Why remove all this code? The `if 0 ` part I understand why. However, the `response =...` isn't that used for the endpoint to get data from the noun project based on the API key?

Additionally, `logger.info...` I think I understand why that needs to be removed since this gets the request and prints it to the logs (for debugging purposes)

Reply:
Nevermind I think I know why we need to remove all of this because the blacklist function will send a get request on behalf of the app.